nimbus/src/enrollment.rs

```diff
@@ -484,8 +488,9 @@ pub fn get_enrollments(db: &Database) -> Result<Vec<EnrolledExperiment>> {
             ..
         } = &enrollment.status
         {
-            if let Some(experiment) =
-                db.get::<Experiment>(StoreId::Experiments, &enrollment.slug)?
+            if let Some(experiment) = db
+                .get_store(StoreId::Experiments)
+                .get::<Experiment, _>(reader, &enrollment.slug)?
```

Comment:
Another function-signature annoyance related to the `Readable` trait- `SingleStore::get` now takes two generic type parameters (one for the data type, one for the concrete transaction type implementing `Readable`) so we have to instruct the turbofish to infer the type of the second.

I think this is actually an opportunity for further future cleanup - we *know* that the `StoreId::Experiments` store is always going to want to return items of type `Experiment`, so we should be able to avoid having to turbofish here at all, perhaps by making explicit types like `ExperimentsStore` and `EnrollmentsStore` and putting the type of contained data as an associated type. But anyway, that's for the future.